lib/licensee/projects/fs_project.rb

```diff
@@ -10,7 +10,11 @@
 module Licensee
   module Projects
     class FSProject < Licensee::Projects::Project
-      def initialize(path, **args)
+      # rubocop:disable UnusedMethodArgument
+      def initialize(path, revision: nil, **args)
+        # (revision isn't used by fs_project)
+        # rubocop:enable UnusedMethodArgument
```

Comment:
Rather than silently ignoring the argument if it's passed, would it make more sense to raise an ArgumentError here to inform the user?